### PR TITLE
Scripts/Spells : Paladin_Spells Cleanup(Small)

### LINKED ISSUE #18353: [Master] Paladin Ability: Templar's Verdict (85256)
[//]: # (***************************************************************)
[//]: # (** DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED **)
[//]: # (***************************************************************)

**Description:**
Using Templar's Verdict results in an animation but no effect.
**Current behaviour:** (Tell us what happens.)
Templar's Verdict shows animation and expends holy charge (power), beginning auto attack vs. target enemy but dealing no damage itself.
http://imgur.com/a/qttVW
**Expected behaviour:** (Tell us what should happen instead.)
Templar's Verdict deals damage to target according to tooltip.
**Steps to reproduce the problem:**

1. Learn Templar's Verdict on paladin player
2. Gather 3 Holy Charge
3. Use Templar's Verdict on enemy.
4. See animation and holy charge diminish but no effect.

**Branch(es):** 3.3.5 / master (Tell us which branch(es) this issue affects.)
master
**TC rev. hash/commit:**  
9b006e511bf40020e1660adf290ceb6ed6f4fd86
**TDB version:**  
703.00
**Operating system:**  
Windows 8.1

[//]: # (This template is for problem reports. For other types of report, edit it accordingly.)
[//]: # (If this is a crash report, include the crashlog with https://gist.github.com/)
[//]: # (For fixes containing C++, create a Pull Request.)

### LINKED ISSUE #18384: [Master] Paladin Spells: Lay on Hands (633)/Divine Shield (642)/Blessing of Protection (1022) - No forbearance (25771)
[//]: # (***************************************************************)
[//]: # (** DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED **)
[//]: # (***************************************************************)

**Description:**
http://www.wowhead.com/spell=25771/forbearance
http://www.wowhead.com/spell=633/lay-on-hands
http://www.wowhead.com/spell=642/divine-shield
http://www.wowhead.com/spell=1022/blessing-of-protection
**Current behaviour:** (Tell us what happens.)
Casting lay on hands, divine shield, or blessing of protection does not apply debuff forbearance. Therefore, the paladin can cast all three in rapid succession
http://imgur.com/a/akmkU
**Expected behaviour:** (Tell us what should happen instead.)
Casting one of these three spells applies forbearance, preventing being target from the other two for 30 seconds.
**Steps to reproduce the problem:**

1. .learn 633
2. .learn 642
3. .learn 1022
4. cast all three in succession

**Branch(es):** 3.3.5 / master (Tell us which branch(es) this issue affects.)
master
**TC rev. hash/commit:**  
fb4097cc4a95c0c8ac9973ec1557b1ceb1cfdc65
**TDB version:**  
703.00
**Operating system:**  
Windows 8.1

[//]: # (This template is for problem reports. For other types of report, edit it accordingly.)
[//]: # (If this is a crash report, include the crashlog with https://gist.github.com/)
[//]: # (For fixes containing C++, create a Pull Request.)

### LINKED ISSUE #19670: Core/Script Spell Talent Final Stand presenting problems.
**Description:**

The passive power Final Stand of the paladin "Protection" it is presenting problems.

**Current behaviour:**

When using Divine Shield this it doesn't provoke the enemies around.

**Expected behaviour:**

When using Divine Shield this it should cause during 8 seconds to the enemies in an area of 15 yards.

**Steps to reproduce the problem:**

1. create a player class paladin
2. reach level 10 and to choose the specialization of "Protection"
3. reach level 75 and to choose Final Stand as talent point
4. use Divine Shield around enemies

**Branch(es):**

Master

**TC rev. hash/commit:** 

7c66e7bbc7a401e7a2d8f2bdcead6531678262ea

**TDB version:**  7.2.0+updates

**Operating system:** Windows 10

Description in wowhead http://www.wowhead.com/spell=204077/final-stand

### LINKED ISSUE #19690: Core/Script Debuff Forbearance is not working at all.
**Description:**

The Paladin Debuff called Forbearance is not working

**Current behaviour:**

When you use the Divine Shield or Blessing of Protection you don´t have any debuff on you.

**Expected behaviour:**

When you use the spell Divene Shield or Blessing of Protection on you should have a debuff called Forbearance that not let you use the Divine Shield or Blessing of Protection on you for 30.3 seconds.

**Steps to reproduce the problem:**

1. Create a Paladin
2. Reach level 18
3. Use the spell Divine Shield

**Branch(es):**

Master

**TC rev. hash/commit:** 

7c66e7bbc7a401e7a2d8f2bdcead6531678262ea

**TDB version:**  7.2.0+updates

**Operating system:** Windows 10

Description in wowhead http://www.wowhead.com/spell=25771/forbearance

### LINKED ISSUE #19692: Core/Script Paladin Retribution Talent Blinding Light.
**Description:**

The paladin Talent Blinding Light of the Retribution specialization is not working

**Current behaviour:**

When you select the talent Blinding Light and then use the spell that has the same name near by a target you are doing the animation but nothing is happen

**Expected behaviour:**

When you use the spell Blinding Light you has to Emits dazzling light in all directions, blinding enemies within 10 yars, dealing holy damage and causing them to wander disoriented for 6 seconds. Non-holy damage will break the disorient effect

**Steps to reproduce the problem:**

1. Create a paladin
2. Reach level 10 and select Retribution Specialization
3. Reach level 45 and select the talent Blinding Light
4. Use the spell Blinding Light near by a enemy

**Branch(es):**

Master

**TC rev. hash/commit:** 

7c66e7bbc7a401e7a2d8f2bdcead6531678262ea

**TDB version:**  7.2.0+updates

**Operating system:** Windows 10

Description in wowhead http://www.wowhead.com/spell=115750/blinding-light

### LINKED ISSUE #19700: Core/Script Spell Templar´s Veredict has an issue.
**Description:**

The Paladin spell Templar´s Veredict is not working

**Current behaviour:**

When you use the Templar´s Veredict over a target the damage that you are doing is 0

**Expected behaviour:**

When you use on a target with the 3 charges of Holy Power should hit the target doing a holy damage of 107 (Level 10)

**Steps to reproduce the problem:**

1. Create a Paladin player
2. Reach Leve 10
3. Use the Crusader Striker 3 times to generate 3 charges of Holy Power
4. Use the Templar´s Veredict on a targert

**Branch(es):**

Master

**TC rev. hash/commit:** 

7c66e7bbc7a401e7a2d8f2bdcead6531678262ea

**TDB version:**  7.2.0+updates

**Operating system:** Windows 10

Description in wowhead http://www.wowhead.com/spell=85256/templars-verdict

### PR DESCRIPTION
**Pull request by Hadifaren**
**Changes proposed:**
removed some spells and added some new spells

**Target branch(es):** 3.3.5/master
- [x] master

**Issues addressed:** Closes #  (insert issue tracker number)

**Tests performed:** (Does it build, tested in-game, etc.)
new scripts tested and works in game

**Known issues and TODO list:** (add/remove lines as needed)

- [ ] 
- [ ]

### CODE REVIEW

File: src/server/scripts/Spells/spell_paladin.cpp [hunk 1]
```diff
@@ -33,204 +33,41 @@
...
+    SPELL_PALADIN_TEMPLAR_VERDICT_HOLY_DAMAGE    = 138165
 };
 
 enum PaladinSpellVisualKit
 {
     PALADIN_VISUAL_KIT_DIVINE_STORM = 73892
 };
 
-/*
-// 31850 - Ardent Defender
```
Comment (joschiwald) [inline src/server/scripts/Spells/spell_paladin.cpp]: spell still exists

Comment (Hadifaren) [reply src/server/scripts/Spells/spell_paladin.cpp]: removed because it has been surrounded for long time and nobody want to fix it ^_^

Comment (joschiwald) [reply src/server/scripts/Spells/spell_paladin.cpp]: that is not the right way

Comment (Hadifaren) [reply src/server/scripts/Spells/spell_paladin.cpp]: you right i put it back

File: src/server/scripts/Spells/spell_paladin.cpp [hunk 2]
```diff
@@ -274,66 +111,29 @@ class spell_pal_avenging_wrath : public SpellScriptLoader
         }
 };
 
-// 37877 - Blessing of Faith
```
Comment (joschiwald) [inline src/server/scripts/Spells/spell_paladin.cpp]: also still exists

**Comment by joschiwald** [comment]
when adding new scripts or changing existing one, create one pr for each spellscript, dropping multiple obsolete spellscripts in one pr is fine

File: src/server/scripts/Spells/spell_paladin.cpp (hunk 2 above)
Comment (Hadifaren) [reply src/server/scripts/Spells/spell_paladin.cpp]: this spell is for item 30841 need to be moved to item_script or here is good?

**Comment by Hadifaren** [comment]
i can remove this pull request and made another one , if u want

**Pull request closed without merging**